en/Extending Obsidian/Obsidian URI.md

```diff
@@ -21,140 +30,239 @@ The `action` parameter is the action that you would like to perform. Available a
+
+### `path`
+
+An absolute file system path to a file on your computer.
+
+- Using this parameter will override both `vault` and `file`.
+- The app will search for the most specific vault that contains the specified file path.
+- The remainder of the path after the vault root replaces the `file` parameter.
+
+For example, if you have a vault at `/home/user/my vault` and use `path=/home/user/my vault/folder/note`, this is equivalent to setting `vault=my vault` and `file=folder/note`.
 
 ## Open note
 
 The `open` action opens an Obsidian vault, or opens a file within that vault.
 
+### Parameters
+
+The `open` action uses the [[#Common parameters]] (`vault`, `file`, `path`).
+
 ### Examples
 
 - `obsidian://open?vault=my%20vault`
-  This opens the vault `my vault`. If the vault is already open, focus on the window.
+
+  Opens the vault `my vault`. If the vault is already open, focuses on the window.
+
 - `obsidian://open?vault=ef6ca3e3b524d22f`
-  This opens the vault identified by the ID `ef6ca3e3b524d22f`.
+
+  Opens the vault identified by the ID `ef6ca3e3b524d22f`.
+
 - `obsidian://open?vault=my%20vault&file=my%20note`
-  This opens the note `my note.md` in the vault `my vault`, assuming the file exists.
+
+  Opens the note `my note.md` in the vault `my vault`, assuming the file exists.
+
 - `obsidian://open?path=%2Fhome%2Fuser%2Fmy%20vault%2Fpath%2Fto%2Fmy%20note`
-  This will look for any vault that contains the path `/home/user/my vault/path/to/my note`. Then, the rest of the path is passed to the `file` parameter. For example, if a vault exists at `/home/user/my vault`, then this would be equivalent to `file` parameter set to `path/to/my note`.
 
+  Looks for any vault that contains the path `/home/user/my vault/path/to/my note`. The rest of the path is passed to the `file` parameter. For example, if a vault exists at `/home/user/my vault`, this is equivalent to setting `file=path/to/my note`.
 
 > [!tip] Open a heading or block
-> With proper URI encoding, you can navigate to a heading or block within a note. `Note%23Heading` would navigate to the heading called "Heading", whereas `Note%23%5EBlock` would navigate to the block called "Block".
+> With proper URI encoding, you can navigate to a heading or block within a note. `Note%23Heading` navigates to the heading called "Heading", whereas `Note%23%5EBlock` navigates to the block called "Block".
 
-### Parameters
+### Add content to an existing note
+
+You can add content to an existing note when opening it using the `prepend` or `append` parameters. This is useful for automation workflows where you want to add timestamped entries, append daily logs, or prepend important reminders to existing notes.
+
+#### Parameters
+
+- `prepend` — Content to add to the top of the file. The operation will attempt to merge [[Properties|properties]] if they exist.
+- `append` — Content to add to the end of the file. The operation will attempt to merge [[Properties|properties]] if they exist.
```

Comment:
How does the merging of properties work? Without a link or further explanation I'd never use this in fear that it completely destroys what I have.